app/grandchallenge/jqfileupload/views.py

```diff
@@ -93,3 +94,11 @@ def _handle_file(self, uploaded_file):
             "total_size": total_size,
             "user_pk_str": self.user_pk_str,
         }
+
+    @action(detail=False, methods=["get"])
+    def get_current_file_size(self, request):
+        client_id = request.GET.get("file", None)
+        files = StagedFile.objects.filter(client_id=client_id).aggregate(
+            Max("end_byte")
+        )
```

Comment:
Is this correct? What happens if the chunks arrive out of order?

Reply:
Good point, I guess we also need to check for any gaps in the byte sequence and return the `end_byte` for the largest uninterrupted sequence of chunks.